models/issue.go

```diff
@@ -1652,6 +1652,8 @@ func GetUserIssueStats(opts UserIssueStatsOptions) (*IssueStats, error) {
 	var repoCond = builder.NewCond()
 	if opts.RepoSubQuery != nil {
 		repoCond = builder.In("issue.repo_id", opts.RepoSubQuery)
+	} else {
+		repoCond = builder.Expr("0=1")
 	}
```

Comment:
```
	var repoCond = builder.NewCond()
	if opts.RepoSubQuery != nil {
		repoCond = builder.In("issue.repo_id", opts.RepoSubQuery)
	} else {
		repoCond = builder.Expr("0=1")
	}
```
I'd move this block inside the `switch/case FilterModeAll:`. Having it here is confusing.

Reply:
But `stats.YourRepositoriesCount` on line 1742 will also use that.

Reply:
Sorry, I missed that.